AmplifyPlugins/Logging/Sources/AWSCloudWatchLoggingPlugin/AWSCloudWatchLoggingSessionController.swift

```diff
@@ -85,9 +85,12 @@ final class AWSCloudWatchLoggingSessionController {
     }
 
     func enable() {
-        updateSession()
-        updateConsumer()
-        connectProducerAndConsumer()
+        Task {
```

Comment:
Can we think of anything that might break from changing this behaviour? Previously calling `enable()` basically guaranteed that the session and consumer were updated and linked, but now that's no longer the case as it's an async operation.

Reply:
this should be okay to be async since the log producers and consumer sub-components are async already